src/library/export/dlglibraryexport.cpp

```diff
@@ -80,25 +100,25 @@ DlgLibraryExport::DlgLibraryExport(
     connect(m_pWholeLibraryRadio,
             &QRadioButton::clicked,
             this,
-            [this]() { m_pCratesList->setEnabled(false); });
-    m_pCratesRadio = make_parented<QRadioButton>(tr("Selected crates"), this);
-    connect(m_pCratesRadio,
+            [this]() { m_pCratesList->setEnabled(false); m_pPlaylistsList->setEnabled(false); });
+    m_pCratesAndPlaylistsRadio = make_parented<QRadioButton>(tr("Selected crates/playlists"), this);
+    connect(m_pCratesAndPlaylistsRadio,
             &QRadioButton::clicked,
             this,
-            [this]() { m_pCratesList->setEnabled(true); });
+            [this]() { m_pCratesList->setEnabled(true); m_pPlaylistsList->setEnabled(true); });
 
     // Button to allow ability to browse for the export directory.
     auto pExportDirBrowseButton = make_parented<QPushButton>(tr("Browse"), this);
     connect(pExportDirBrowseButton,
             &QPushButton::clicked,
             this,
             &DlgLibraryExport::browseExportDirectory);
-    auto pExportDirLayout = make_parented<QHBoxLayout>(this);
+    auto pExportDirLayout = std::make_unique<QHBoxLayout>();
     pExportDirLayout->addWidget(m_pExportDirectoryTextField);
     pExportDirLayout->addWidget(pExportDirBrowseButton);
 
-    auto pFormLayout = make_parented<QFormLayout>(this);
-    pFormLayout->addRow(tr("Export directory"), pExportDirLayout);
+    auto pFormLayout = std::make_unique<QFormLayout>();
```

Comment:
The prior code failed debug assertions... the layout is created initially unparented (so not using `make_parented<T>()`), and is then added as a layout later, which re-parents the layout at the same time.

Prior code approach was introduced here:
https://github.com/mixxxdj/mixxx/commit/907d5fdb91a0b3bc9cd911781329ae9eb50d8f74